### PR TITLE
Add api call for a page block

### PR DESCRIPTION
**Pull request by carakas**
## Type

- Feature

## Pull request description

Adds an api call to get the content that the widget or action would've sent to the twig template

example call
`/api/pages/en/1/page-block/9.json`

There is a lot of stuff in the way that we render things that can be refactored but that is out of scope for this pr

### CODE REVIEW

**Comment by StijnVrolijk** [comment]
@carakas why do we need this feature? How is the request built up? What is `9`? Is it the revision ID?

**Comment by carakas** [comment]
9 is the module extra id and 1 is the page.
It is used to get the data that the widget would pass to the template if that module extra was rendered on that page. If you want to build a headless site you need this to get the data from the widget/action that is added in the backend to the page

File: src/Backend/Modules/Pages/Domain/ModuleExtra/ModuleExtraType.php [hunk 1]
```diff
@@ -56,4 +71,5 @@ public function jsonSerialize(): string
     {
         return (string) $this;
     }
+
```
Comment (StijnVrolijk) [inline src/Backend/Modules/Pages/Domain/ModuleExtra/ModuleExtraType.php]: This whitespace should be removed according to PSR-2

**Comment by StijnVrolijk** [comment]
@carakas how do you plan on fetching something like a blogpost that gets its information from the URL? e.g. `/en/blog/detail/my-blog-post` won't work with this implementation because the block on the detail page requires `my-blog-post` to know what to fetch.

**Review by StijnVrolijk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**